cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/PageNotification/PageNotification.tsx

```diff
@@ -0,0 +1,43 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { CdsAlert, CdsAlertGroup } from '@cds/react/alert';
+import { StatusTypes } from '@cds/core/internal';
+
+export const NotificationStatus = {
+    DANGER: 'danger',
+    INFO: 'info',
+    SUCCESS: 'success',
+    WARNING: 'warning',
+};
+
+export interface Notification {
+    status: string;
```

Comment:
Does it make sense to declare this as a StatusTypes (since later we're using `as StatusTypes`)?